cmd/bnsd/app/testdata/fixtures/app.go

```diff
@@ -34,7 +35,12 @@ func NewApp() *AppFixture {
 }
 
 func (f AppFixture) Build() abci.Application {
-	myApp, err := app.GenerateApp("", log.NewNopLogger(), true)
+	opts := &server.Options{
+		Home:"",
+		Logger: log.NewNopLogger(),
+		Debug: true,
```

Comment:
Why is there no MinFee here. Even if it is just coin.Coin{} ?

You also include other zero values (`Home: "", Debug: false` other places)